src/ListActions/ListActions.js

```diff
@@ -484,7 +484,9 @@ var ListActions = module.exports = kind(
 		var listAction, i;
 
 		this.listActionComponents = [];
-		this.$.listActions.destroyClientControls();
+		if (this.generated) {
```

Comment:
This can't be taken because `generated` isn't a safe-guard for actual client controls existence. I'm not sure that this bug is actually a bug? It is harmless/optimizable function calls that are essentially a nop when no controls exist. Why is this even being considered?